config/clouddriver.yml

```diff
@@ -101,7 +101,9 @@ dockerRegistry:
     - name: ${providers.dockerRegistry.primaryCredentials.name}
       address: ${providers.dockerRegistry.primaryCredentials.address}
       username: ${providers.dockerRegistry.primaryCredentials.username:}
-      passwordFile: ${providers.dockerRegistry.primaryCredentials.passwordFile}
+      passwordFile: ${providers.dockerRegistry.primaryCredentials.passwordFile:}
+      repositories:
+        - ${providers.dockerRegistry.primaryCredentials.repository}
```

Comment:
What about if you do this in clouddriver instead of list it on the next line?

> repositories: ${providers.dockerRegistry.primaryCredentials.repository}

This one seems to work for me in both cases (it queries the catalog if the repository isn't specified and has a list of 1 repository if I do specify it)